### PR TITLE
expect-expect

### PR DESCRIPTION
**Pull request by deecewan**
Based on TandaHQ/payaus#4055, I've created a rule to enforce the use of expect.

If you can think of a usage of expect that I haven't covered, let me know and I'll add a test.

The case where this won't work is if you do something like this:

```js
it('should pass', () => {
  someArray.forEach(a => {
    if (someCondition) { throw new Error('Should have passed'); }
  });
});
```

But, I feel like this is so rarely the case that we should explicitly disable when it is the case.

### CODE REVIEW

File: docs/rules/expect-expect.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Expect that expect is called in jest tests (expect-expect)
+
+If people don't call expect in a test, it's not going to work as expected
```
Comment (ghiculescu) [inline docs/rules/expect-expect.md]: typo: `If`

File: lib/rules/expect-expect.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+
+    //----------------------------------------------------------------------
+    // Public
+    //----------------------------------------------------------------------
+
+    return {
+      // give me methods
+      CallExpression(node) {
+        debugger;
```
[the remark this answers is not in the document]
Comment (deecewan) [reply lib/rules/expect-expect.js]: whoooooops

**Comment by deecewan** [comment]
But the tests pass?

**Comment by deecewan** [comment]
@daveallie do you mean in the index?

**Comment by daveallie** [comment]
yeah, I released a fix and published 0.0.11

https://github.com/TandaHQ/eslint-plugin-tanda/commit/48856dd0b036685895e7c99a3613b018c6f1f4bf

**Pull request merged**